src/studio/src/designer/frontend/dashboard/features/dashboard/OrgReposList.tsx

```diff
@@ -11,7 +11,10 @@ import { useGetUserStarredReposQuery } from 'services/userApi';
 import { useAugmentReposWithStarred } from './hooks';
 import { getUidFilter, getReposLabel } from './utils';
 
+const rowsPerPageOptions = [5, 10, 20]
```

Comment:
```suggestion
const rowsPerPageOptions = [5, 50, 100, 1000]
```

Is performance actually so bad that we need to limit the number of elements on a page?

Reply:
Its not about the performance really, but what the need is. Discussed with UX and decided to increase/add more options. 

We cant go to 1000, because of limitations in the component we use.

Thanks for the input 🙂

Reply:
100 is a lot better than 20. Thanks!